redux/alertsSlice.ts

```diff
@@ -0,0 +1,64 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from './store';
+import { Alert } from '../types';
+
+export type AlertMessage = {
+  id: number;
+} & Alert;
+
+interface MessagesState {
+  id: number;
+  alertsList: AlertMessage[];
+}
+
+let alertID = 0;
+
+const mockedData = {
+  alerts: [
+    {
+      id: alertID++,
```

Comment:
Instead of creating an id that is a plain number, you can use `nanoid` from `redux/toolkit
````
import { nanoid } from '@reduxjs/toolkit'
{id: nanoid(), ...}
```